sdk/Sdk/FunctionMetadataGenerator.cs

```diff
@@ -804,8 +806,75 @@ private static bool SupportsDeferredBinding(CustomAttribute attribute)
                 return false;
             }
 
-            return typeDefinition.CustomAttributes
-                                 .Any(a => string.Equals(a.AttributeType.FullName, Constants.SupportsDeferredBindingAttributeType, StringComparison.Ordinal));
+            // checking attributes advertised by the binding attribute
+            foreach (CustomAttribute bindingAttribute in typeDefinition.CustomAttributes)
+            {
+                if (string.Equals(bindingAttribute.AttributeType.FullName, Constants.InputConverterAttributeType, StringComparison.Ordinal))
+                {
+                    // InputConverterAttribute will have supported converter type
+                    foreach (var customAttribute in bindingAttribute.ConstructorArguments)
```

Comment:
InputConverterAttribute ctor just takes a single parameter "converterType" right? The fact that you're naming this "customAttribute" here and foreaching confuses me.

Reply:
Yes, updated the name to converterType instead of customAttribute. Foreaching only because at this point, we don't know that this attribute has how many constructor parameters.